src/components/MediaSettings/VideoBackgroundEditor.vue

```diff
@@ -110,10 +140,111 @@ export default {
 		},
 	},
 
+	async mounted() {
+		// userRoot path
+		const userRoot = '/files/' + this.$store.getters.getUserId()
+
+		// Relative background folder path
+		const relativeBackgroundsFolderPath = this.$store.getters.getAttachmentFolder() + '/Backgrounds'
+
+		// Absolute backgrounds folder path
+		const absoluteBackgroundsFolderPath = userRoot + relativeBackgroundsFolderPath
+
+		// Create the backgrounds folder if it doesn't exist
+		if (await client.exists(absoluteBackgroundsFolderPath) === false) {
+			await client.createDirectory(absoluteBackgroundsFolderPath)
+		}
+
+		// Create picker
+		picker = getFilePickerBuilder(t('spreed', 'File to share'))
+			.setMultiSelect(false)
+			.setModal(true)
+			.startAt(relativeBackgroundsFolderPath)
+			.setType(1)
+			.allowDirectories(false)
+			.build()
+
+		this.loadBackground()
+	},
```

Comment:
**suggestion:** load background first.

Currently backgrounds are loaded from local storage only after the async check of the folder existence. It may be not fast on a slow connection. We can load them with `this.loadBackground()` before this check. Then it won't be blocked.